### ISSUE TITLE
Apply the PRS trained  on a test set

### ISSUE THREAD

**Issue by emiled16** [choishingwan/PRSice#239]
I am using PRSice to compute the PRS over a train set and want to use the coefficient  used on the train set to apply it on another set which I will call test set.

Once I compute the PRS I get a file called  analysis.prsice
This is the few first lines

___________________________________________________________________________________________
Pheno   Set     Threshold       R2      P       Coefficient     Standard.Error  Num_SNP
-       Base    5e-08   0.00550157      0.0659631       -21.0513        11.4492 43
-       Base    5.005e-05       0.00396637      0.113759        -61.8545        39.1108 184
-       Base    0.00010005      0.00236122      0.220463        -68.9527        56.2742 286
-       Base    0.00015005      0.00113035      0.394503        -59.4058        69.7677 382
-       Base    0.00020005      0.00072253      0.495129        -55.2605        81.0064 471
-       Base    0.00025005      0.000334179     0.642121        -41.5674        89.4432 538
-       Base    0.00030005      7.5306e-05      0.825111        -21.391 96.8021 601
-       Base    0.00035005      3.95773e-06     0.959527        5.25804 103.612 668
-       Base    0.00040005      0.000175506     0.735018        37.723  111.455 743
___________________________________________________________________________________________

I thought that  I could apply the prs on another dataset by multiplying the coefficient in the result by the allelic dosage of the corresponding SNPs. However, I am not sure how to know which are the corresponding SNPs of the coefficients. 
Also, if there was a command that does that and it would be easier, could you please tell me how to do it?
Thanks

**Comment by choishingwan** [comment]
The best way to do this is to obtain the best p-value threshold from the test case (output in the .summary file)

Then rerun the analysis on an independent validation set with the same summary statistics and with the following options

`--bar-levels <best p-value threshold> --fastscore --no-full`

(if you don't have phenotype for the validation dataset, then also add `--no-regress`)

This will generate the PRS on the validation dataset for you.

**Comment by emiled16** [comment]
Thank you  Shing Wan Choi. I appreciate it
